src/plugins/azgenerator/TemplateAzureCliActions.ts

```diff
@@ -73,7 +77,12 @@ export function GenerateAzureCliActions(model: CodeModelAz) : string[] {
                                                         continue;
                                                     }
                                                     output.push("            " + ifkv + " kl == '" + model.MethodParameter_NameAz + "':");
-                                                    output.push("                d['" + model.MethodParameter_NamePython + "'] = v");
+                                                    if (model.MethodParameter_IsArray) {
+                                                        output.push("                d['" + model.MethodParameter_NamePython + "'] = v");
+                                                    }
+                                                    else {
+                                                        output.push("                d['" + model.MethodParameter_NamePython + "'] = v[0]");
+                                                    }
```

Comment:
does this mean you can only pass one value to an array?

Reply:
refer to Line 80, if the schema is not an array, it will go to this line.

Reply:
I see